LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Greene Laboratory
```

Comment:
Do you need to include Chris Polis here? Is any of the code from that library, or is this a new implementation of the same ideas/approaches?

Reply:
None of the code is copy-pasted directly, but I did look closely at his code while writing my own. I do mention Chris Polis in the readme. Not sure if we need to put it in the license as well. Note that the license was just an automatically generated one by Github.

Reply:
Ok - I didn't realize from how it was described as adapted that none of the code was taken from there. Maybe "Inspired"?